### DISCUSSION TITLE
Weird sounds and game sped up like 2000x

### DISCUSSION THREAD

**Discussion by Nexenek** [ppy/osu#23102] [Q&A]
### Type

Game behaviour

### Bug description

I alt-tabbed from my game to check discord and this happened next

(restart doesn't fix the issue)

### Screenshots or videos

https://user-images.githubusercontent.com/94640178/229107358-72e2ae93-d237-4bcf-ab30-fff2c8d17056.mp4

https://user-images.githubusercontent.com/94640178/229109019-e822a09c-01e7-45be-9e16-1be22c7b4d96.mp4

### Version

2023.326.1

### Logs

[database.log](https://github.com/ppy/osu/files/11121256/database.log)
[input.log](https://github.com/ppy/osu/files/11121257/input.log)
[network.log](https://github.com/ppy/osu/files/11121258/network.log)
[performance.log](https://github.com/ppy/osu/files/11121259/performance.log)
[performance-draw.log](https://github.com/ppy/osu/files/11121260/performance-draw.log)
[performance-update.log](https://github.com/ppy/osu/files/11121261/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/11121262/runtime.log)

**Comment by Nexenek** [comment]
Just turned on auto-play and when I started the map it fixed itself?

**Comment by Nexenek** [comment]
[database.log](https://github.com/ppy/osu/files/11121338/database.log)
[input.log](https://github.com/ppy/osu/files/11121339/input.log)
[network.log](https://github.com/ppy/osu/files/11121340/network.log)
[performance.log](https://github.com/ppy/osu/files/11121341/performance.log)
[performance-draw.log](https://github.com/ppy/osu/files/11121342/performance-draw.log)
[performance-update.log](https://github.com/ppy/osu/files/11121343/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/11121344/runtime.log)


(logs after fix)

**Comment by peppy** [comment]
Likely linux related. If someone repro's on windows I'm willing to look into it further.

**Comment by soopax** [comment]
This sometimes happens to me. It can take very long to fix itself. 
OS is Debian GNU/Linux 11 (bullseye) and I'm using GNOME 3.38.6
